task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: fbc-validation
+spec:
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name
+  results:
+    - name: HACBS_TEST_OUTPUT
+      type: string
+  steps:
+    - name: check-labels-and-binaries
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      workingDir: /work
+      securityContext:
+        runAsUser: 1000
+      env:
+        - name: HOME
+          value: /work
+      resources:
+        limits:
+          memory: 4Gi
+          cpu: 2
+        requests:
+          memory: 512Mi
+          cpu: 10m
+      script: |
+        ### Check if label exists > If yes try to extract binaries with configs > check binaries functionality > check opm validate ###
+        conffolder=$(skopeo inspect --format '{{ 1 | or (index .Labels "operators.operatorframework.io.index.configs.v1") }}' docker://"$(params.IMAGE_URL)")
```

Comment:
I believe we'll be running the [sanity-inspect-image](https://github.com/redhat-appstudio/build-definitions/blob/main/task/sanity-inspect-image/0.1/sanity-inspect-image.yaml) task as part of the FBC pipeline, so the skopeo output would already be available to use here

Reply:
removed